distribution/sender/routes/sender-activemq-consumer.xml

```diff
@@ -0,0 +1,93 @@
+        <from uri="activemq:openmrs.sync.{{db-sync.senderId}}?connectionFactory=activeMqConnFactory&amp;acknowledgementModeName=CLIENT_ACKNOWLEDGE&amp;messageListenerContainerFactory=customMessageListenerContainerFactory&amp;asyncStartListener=true" />
+
+        <log message="Received message -> ${body}" />
+        
+        <setProperty name="requestUuid">
+            <jsonpath suppressExceptions="true">$.requestUuid</jsonpath>
+        </setProperty>
+        <setProperty name="messageUuid">
+            <jsonpath suppressExceptions="true">$.messageUuid</jsonpath>
+        </setProperty>
+        
+        <choice>
+            <when>
+                <simple>${exchangeProperty.requestUuid} != null</simple>
+                
+                <setProperty name="requestTableName">
+                    <jsonpath>$.tableName</jsonpath>
+                </setProperty>
+                <setProperty name="requestIdentifier">
+                    <jsonpath>$.identifier</jsonpath>
+                </setProperty>
+                <setProperty name="requestToSave">
+                    <spel>
+                        #{new org.openmrs.eip.app.management.entity.SenderSyncRequest()}
+                    </spel>
+                </setProperty>
+                <script>
+                    <spel>
+                        #{getProperty('requestToSave').setTableName(getProperty('requestTableName'))}
+                        #{getProperty('requestToSave').setIdentifier(getProperty('requestIdentifier'))}
+                        #{getProperty('requestToSave').setRequestUuid(getProperty('requestUuid'))}
+                        #{getProperty('requestToSave').setDateCreated(new java.util.Date())}
+                    </spel>
+                </script>
+                <setBody>
+                    <simple>${exchangeProperty.requestToSave}</simple>
+                </setBody>
+        
+                <log loggingLevel="DEBUG" message="Saving sync request -> ${body}" />
+        
+                <to uri="jpa:SenderSyncRequest" />
+        
+                <log message="Successfully saved sync request" />
+            </when>
+            
+            <when>
+                <simple>${exchangeProperty.messageUuid} != null</simple>
+                
+                <setProperty name="senderSyncResponseModelClass">
+                    <spel>#{T(org.openmrs.eip.app.management.entity.SenderSyncResponseModel)}</spel>
+                </setProperty>
+                
+                <setProperty name="senderSyncResponseModel">
+                    <method beanType="org.openmrs.eip.component.utils.JsonUtils" method="unmarshal(${body}, ${exchangeProperty.senderSyncResponseModelClass})" />
+                </setProperty>
+                
+                <setProperty name="dateSent">
+                    <method beanType="org.openmrs.eip.component.utils.DateUtils" method="localDateTimeToDate(${exchangeProperty.senderSyncResponseModel.dateSent})" />
```

Comment:
Why do you need to convert this date, just copy the already converted dateSent value from the senderSyncResponseModel you just unmarshalled above